src/services/graphql/fragments/Admin/index.js

```diff
@@ -65,6 +65,15 @@ export const blacklist = `fragment BlacklistView on BlacklistCollection {
   }
 }`;
 
+export const trustedsources = `fragment TrustedSourcesView on SourceCollection {
```

Comment:
Looks like we both did this (https://github.com/CatalystCode/project-fortis-interfaces/pull/76/files). Sorry for the delay!